### DISCUSSION TITLE
Option to archive entries?

### DISCUSSION THREAD

**Discussion by patrickunterwegs** [TechbeeAT/jtxBoard#189] [Polls]
There was a feature request to be able to archive entries so that they are not visible unless you explicitly include the archive.
The problem is that the standard does not have such a flag/status. But categories could be used - for example a category "archive" could be defined when an entry is is marked as archived. But still this would add a new category to the entry which might not be desired. At the same time users can anyway assign a category and filter by it. Additionally other systems or apps will not know the "archived" category and will not pin it on top.

So the question is if it makes any sense to implement such a feature. What do you think?

**Comment by DickinsonSa** [comment]
Unless it provides any additional functionality, personally I'd be happy to just introduce an 'archive' category - should I need to.

**Comment by rogercreagh** [comment]
of course currently you can only filter to get all of the items tagged as archive, not to exclude them - but that is a more general enhancement mentioned as part of #201
